GSA/made-in-america
# [000]-feature prettier

## Description

using eslint to lint our ts files

## How to test

- fetch changes locally
- `npm install`
- alter a json/ts file
- stage file
- commit file locally to let precommit hook run
- confirm that file gets prettified
- and linter passes

To prepare for code review, please follow the following checks.

- [x] branch has a detailed description of work/updates
- [x] pipeline has been completed successfully
